templates/engage/shared/_header.html

```diff
@@ -20,7 +20,7 @@ <h1 {% if subtitle is none and date is none %}class="u-no-margin--bottom"{% endi
       </div>
     </div>
     {% if image %}<div class="col-5 u-hide--small u-align--center u-vertically-center">
-      <img src="{% if 'http' not in image %}https://assets.ubuntu.com/v1/{% endif %}{{ image }}" width="413" style="max-height: 410px; max-width: 250px;" alt="image for {{ title }}" >
+      <img src="{% if 'http' not in header_image %}https://assets.ubuntu.com/v1/{% endif %}{{ header_image }}" width="413" style="max-height: 410px; max-width: 250px;" alt="image for {{ title }}" >
```

Comment:
The width here is much greater than the inline style restricting the width.

Reply:
I just changed this to work with the new module. There are likely issues.